lib/round.js

```diff
@@ -2,7 +2,8 @@ export class RoundData {
   /** @type {Map<string, string>} */
   #knownStrings
 
-  constructor () {
+  constructor (index) {
+    this.index = index
```

Comment:
AFAICT, we are not using this property anywhere - we always pass the round index in another variable/property alongside the round data. I think we don't need to introduce `this.index`, but it's also possible I have missed something.

WDYT?

Reply:
it's used in round index validation logic: https://github.com/filecoin-station/spark-evaluate/pull/135/commits/505ed331b017141ab0a9121ae41fc0b52e903232

Reply:
I see. Maybe we don't need to keep the index inside round data, but move it to a new variable in `index.js`, alongside `rounds`, `cidsSeen` and `roundsSeen`?

Reply:
What is the motivation for this change?

Reply:
I'm going to merge for now, happy to follow up here